honeybee_radiance/modifierset.py

```diff
@@ -86,6 +89,9 @@ def __init__(self, name, wall_set=None, floor_set=None, roof_ceiling_set=None,
                 If None, it will be the honeybee generic default DoorSet.
             shade_set: An optional ShadeSet object for this ModifierSet.
                 If None, it will be the honeybee generic default ShadeSet.
+            air_wall_modifier: An optional Modifier to be used for all Faces with
```

Comment:
I'm just noting that we discussed this and it unfortunately seems that the term "AirWall" is so ingrained in the modeling culture that trying to change it to something like "AirFace" or "AirSurface" is going to result in more confusion than clarity. I'll note that DesignBuilder tried to coin the term "VirtualPartition" for this case but I worry that using the same terminology here might either be lost on users or it will imply that we are using DesignBuilders odd way of modeling these surfaces in E+ (adding windows to them and not using E+'s AirWallConstruction).